azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/ServicePrincipal.java

```diff
@@ -137,6 +140,7 @@
          * service principal in the cloud, but exposing additional optional inputs to
          * specify.
          */
+        @Beta(SinceVersion.V1_1_0)
```

Comment:
ANother thing related to Beta annotations is that we also need to mark the newly added methods on ServicePrincipal. ServicePrincipal shipped as a model in V1.0.0 marked as `@Beta`, so the newly added methods that expose the new functionality need `@Beta(SinceVersion.V1_1_0)` directly on them. From what i can see, comparing this ServicePrincipal file to one in the sources tagged for V1.0.0, they are `.passwordCredentials()` and `.certificateCredentials()`.